etl/steps/data/garden/un/2024-07-16/migrant_stock.py

```diff
@@ -0,0 +1,362 @@
+import pandas as pd
+from owid.catalog import Table
+from owid.catalog import processing as pr
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+# List of regions (in opposition to countries) in the data set
+REGIONS = [
```

Comment:
Usually, when we define `REGIONS` like this, it's because we will create such region aggregates. In your case, it's different: These are the regions that were already in the data. Just for consistency, and to avoid confusion, you could rename this as `REGIONS_IN_THE_DATA`.
Also, it's not clear to me why they are removed from one of the tables, but not the others. It would be good to clarify here why you are defining this list (and mention that these are the names prior to harmonization).
Finally, you could also explain why you decided to not create your own regions (for continents and income groups) as we usually do.